src/services/services.js

```diff
@@ -0,0 +1,67 @@
+import firebase, { db } from './firebase';
+
+export const isCorrectEmail = (email) => {
+  // eslint-disable-next-line no-useless-escape
+  const regExp = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
move it to global scope

Reply:
could you explain this? i removed eslint exception
```suggestion
export const isCorrectEmail = (email) => {
  const regExp = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
  return regExp.test(email);
};
```

Reply:
it's a good approach to save constant variables outside the methods

Reply:
Thx, now I understand.